### DISCUSSION TITLE
What Approach / Methods Should I Use for Predicting Rice Yield?

### DISCUSSION THREAD

**Discussion by KanataEXE** [community/community#21509] [Other Feature Feedback, Questions, & Ideas]
<p>I have a personal project. The task is to predict the next year rice yield.</p>
<p>I have 3 variables: Rice Yield (1995 - 2019), Productivity (2014-2019), and Harvested Area (2014 - 2019).</p>
<p>What approach / method should I use?</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/tryarie">@tryarie</a>, in my experience, this sounds like a classic linear regression problem. I made my own linear regression setup for one of my internships where I needed to use a bunch of different inputs to predict what the wind speed would be in 20 minutes. </p>
<p>I suggest checking out this example which is the one that I used to make my linear regression setup: <a href="https://github.com/FluxML/model-zoo/blob/master/other/housing/housing.jl" rel="nofollow noopener">https://github.com/FluxML/model-zoo/blob/master/other/housing/housing.jl</a></p>

**Comment by KanataEXE** [comment]
<p>Does that mean I have to use multiple linear regression?</p>
<p>Addendum 1:<br>
I went to search for more data and acquired productivity (1995 - 2019) and harvested area (1995 - 2019).</p>
<p>Addendum 2:</p>
<p>Also, the data is actually per province.</p>
<p>Addendum 3:</p>
<p>On second thought, never mind. I think I know how to do it.</p>
<p>Thank you for helping me.</p>
